LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Services/EmailService.swift

```diff
@@ -67,6 +67,14 @@ struct EmailService: EmailServiceProtocol {
     }
 
     func getEmail(dataBrokerURL: String, attemptId: UUID) async throws -> EmailData {
+
+        /* Ideally we would use staging to get a real email from the endpoint
+         but there's a problem with the token */
+        let dbpSettings = DataBrokerProtectionSettings()
+        if dbpSettings.storedRunType == .integrationTests {
+            return EmailData(pattern: nil, emailAddress: "test@duck.com")
+        }
+
```

Comment:
I unfortunately had to add this at the last minute since suddenly the token isn't working with the email backend. I've made inquiries to try and rectify that, so we may yet be able to remove it. Although if we have to keep it its not horrendous